exercises/Python_Principles_5.ipynb

```diff
@@ -1496,7 +1262,7 @@
     "\n",
     "Consider creating modules when your file is getting larger and harder to reason about the code with lots of functions. Typically speaking a module should have functions that are related together so consider this when you're reading your code. \n",
     "\n",
-    "When you start learning to code, you wont necessarily think in terms of functions and modules and thats okay. As long as you look at your code and start to in small pieces create functions and make sure your code still works. This is a good first step. Always change your code in small steps and rerun it."
+    "When you start learning to code, you wont necessarily think in terms of functions and modules and thats okay. You can always refactor a long piece of code and break it down into smaller pieces. Start to structure the code gradually, and keep making sure your code still works after you write a function or a module. Always change your code in small steps and rerun it."
```

Comment:
The students may not know what refactor means if its there first time doing python, do you think changing the word `refactor` to change` works?

Reply:
Actually only found the first version with "As long as you look at your...". Have you already fixed this?